File: JavaScriptBasics/ConditionalStatements/Exercise/ToyShop.js
function ToyShop(input){

    let tripPrice = parseFloat(input[0]);
    let puzzles = parseFloat(input[1])*2.60;
    let talkingdolls =  parseFloat(input[2])*3;
    let teddies =  parseFloat(input[3])*4.10;
    let minions =  parseFloat(input[4])*8.20;
    let trucks =  parseFloat(input[5])*2;
    let number = parseFloat(input[1])+parseFloat(input[2])+parseFloat(input[3])+parseFloat(input[4])+parseFloat(input[5]);
    let total = puzzles+talkingdolls+teddies+minions+trucks;
    if(number>49){
        total = total*0.75;}
    total = total*0.90;
    if(total>tripPrice){
        console.log("Yes "+(total-tripPrice).toFixed(2)+" lv left.")
    }else{
        console.log("Not enough money! "+(tripPrice-total).toFixed(2)+" lv needed.")
    }
}

File: JavaBasics/ConditionalStatements/Exercise/ToyShop.java
package EntryModule.ConditionalStatements.Exercise;

import java.util.Scanner;
public class ToyShop {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double price = Double.parseDouble(scanner.nextLine());
        int puzzles = Integer.parseInt(scanner.nextLine());
        int dolls = Integer.parseInt(scanner.nextLine());
        int bears = Integer.parseInt(scanner.nextLine());
        int minions = Integer.parseInt(scanner.nextLine());
        int trucks = Integer.parseInt(scanner.nextLine());

        int count = puzzles+dolls+bears+minions+trucks;

        double total = puzzles*2.60+dolls*3+bears*4.10+minions*8.20+trucks*2;
        if(count>=50){total=total*0.75;}

        double result = 0.90*total;
        double da = result-price;
        double ne = price-result;

        if(result>=price){
            System.out.printf("Yes! %.2f lv left.", da);
        }else System.out.printf("Not enough money! %.2f lv needed.", ne);




    }
}
